src/compiler/program.ts

```diff
@@ -2400,6 +2543,70 @@ namespace ts {
         }
     }
 
+    function parseConfigHostFromCompilerHost(host: CompilerHost): ParseConfigHost {
+        return {
+            fileExists: host.fileExists,
+            readDirectory: () => [],
+            readFile: host.readFile,
+            useCaseSensitiveFileNames: host.useCaseSensitiveFileNames()
+        };
+    }
+
+    export function getProjectReferenceFileNames(host: CompilerHost, rootOptions: CompilerOptions): string[] | undefined {

```

Comment:
do we even need this function? why not just expose `walkProjectReferenceGraph`, seems more efficient for most scenarios, and can decide what to do on missing references.

Reply:
Removed